docs/book/v3/features/error-handling.md

```diff
@@ -213,6 +213,23 @@ class LoggingErrorListenerDelegatorFactory
 }
 ```
 
+Then, enable the delegator factory by registering it in the `delegators` element of the relevant [ConfigProvider class](https://docs.laminas.dev/laminas-config-aggregator/config-providers/).
+For example, in `src/App/src/ConfigProvider.php`, update the `getDependencies()` method as follows:
+
+```php
+public function getDependencies(): array
+{
+    return [
+        'delegators' => [
+            ErrorHandler::class => [
+                LoggingErrorListenerDelegatorFactory::class
```

Comment:
```suggestion
            Laminas\Stratigility\Middleware\ErrorHandler::class => [
                Acme\LoggingErrorListenerDelegatorFactory::class
```

Reply:
> …is Acme the standard term used in the Mezzio docs for a hypothetical module?

Only in the documentation of Mezzio itself: https://github.com/search?l=Markdown&q=org%3Amezzio+Acme&type=Code
Therefore, it is correct to modify the path name.

(But in the long run, a better name / example must be found.)